tests/routes/googleLogin.test.js

```diff
@@ -13,14 +13,15 @@ describe('GET /auth/google', () => {
       .request(server)
       .get('/api/v1/auth/google/callback')
       .end((err, res) => {
-        expect(res).to.have.status(200);
-        expect(res.body.user).to.have.property('token');
-        expect(res.body.user.token).to.not.be.null;
-        expect(res.body.user).to.have.property('imageUrl');
-        expect(res.body.user).to.have.property('bio');
-        expect(res.body.user)
-          .to.have.property('email')
-          .that.is.equal('johndoe@gmail.com');
+        expect(res).to.redirect;
+        // expect(res).to.have.status(200);
```

Comment:
Please remove this code if we don't need it anymore

Reply:
Removed, not needed anymore.